### PR TITLE
created tabnav component, pr header tweak etc

### PR DESCRIPTION
**Pull request by joetaylor-harness**

### CODE REVIEW

File: packages/playground/src/components/pull-request/pull-request-conversation-header.tsx [hunk 1]
```diff
@@ -40,45 +40,59 @@ export const PullRequestHeader: React.FC<PullRequestTitleProps> = ({
...
         <div className="flex space-x-2 text-tertiary-background ">
-          <div className="flex items-center align-middle text-center">
+          <div className="flex gap-2.5 items-center align-middle text-center">
             {/* TODO:add status in badge or create a component to handle status */}
             {merged ? (
               <Badge
-                className={`select-none bg-transparent rounded-2xl text-[12px] font-light mr-2 py-1 px-2 leading-none text-ai border-border  hover:bg-inherit`}>
+                disableHover
+                size="default"
+                className={`select-none bg-emerald-300/20 text-emerald-400 font-medium`}>
```
Comment (cjlee01) [inline packages/playground/src/components/pull-request/pull-request-conversation-header.tsx:61]: this should be a purple color since its for merged

**Comment by joetaylor-harness** [comment]
New PR panel using stacked list and accordions, formatted layout, better matching to PP. Have undertaken lots of componentization but there is still lots of css in the code. @cjlee01 I've focused on matching PP so you will want to use this as a guide to inject the data you need. This was quicker than trying to reverse engineer what was already there in some cases. Will look at doing the same kind of guide for the conversation activity blocks in the overview section next.

image

By using accordions inside a stacked list, we are able to match the intention from PP and easily expand rows as necessary

image

**Comment by joetaylor-harness** [comment]
Overview activity timeline:
image

**Comment by joetaylor-harness** [comment]
Added reply box to each overview activity panel:
image

**Review by cjlee01 — APPROVED** [review]

**Pull request merged**